docs/antibodies/README.md

```diff
@@ -0,0 +1,64 @@
+# antibodies
+
+Related files:
+
+- [📝 TSV template](https://raw.githubusercontent.com/hubmapconsortium/ingest-validation-tools/master/docs/antibodies/antibodies-metadata.tsv): Use this to submit metadata.
+- [💻 Source code](https://github.com/hubmapconsortium/ingest-validation-tools/edit/master/src/ingest_validation_tools/table-schemas/antibodies.yaml): Make a PR if this doc should be updated.
+
+## Table of contents
+[`channel_id`](#channel_id)<br>
+[`antibody_name`](#antibody_name)<br>
+[`rrr_id`](#rrr_id)<br>
```

Comment:
```suggestion
[`rr_id`](#181 rr_id)<br>
```

Turns out Marda says it should be rr_id.

Reply:
Why 181? I'll correct rrr to rr, but let me know if the number is there for a reason.